From e0e9002501d8644d386e695883a468b4cb0e96df Mon Sep 17 00:00:00 2001
From: Atanas Filipov <afilipov@mm-sol.com>
Date: Fri, 9 Aug 2013 10:13:45 +0300
Subject: [PATCH 33/56] IFC6410: S4 Pro: Fix the "ext_3p3v" pin number
 according to board schematic

   Added support for Optronics - B101XTN0 LVDS panel

Change-Id: I266855579a1ede42900449a961b73c146d27b686
Signed-off-by: Atanas Filipov <afilipov@mm-sol.com>
Signed-off-by: Suman Kumar <suman@inforcecomputing.com>
---
 arch/arm/mach-msm/board-8064-regulator.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-msm/board-8064-regulator.c b/arch/arm/mach-msm/board-8064-regulator.c
index c27e744..5ff0688 100644
--- a/arch/arm/mach-msm/board-8064-regulator.c
+++ b/arch/arm/mach-msm/board-8064-regulator.c
@@ -582,7 +582,11 @@ apq8064_gpio_regulator_pdata[] __devinitdata = {
 			 * This change is only for a short term workaround.
 			 * We should fix it properly.
 			 */
+#if defined(CONFIG_FB_MSM_LVDS_OPTRONICS)
+			PM8921_MPP_PM_TO_SYS(4), "ext_3p3v", 1),
+#else
 			PM8921_MPP_PM_TO_SYS(3), "ext_3p3v", 1),
+#endif
 };
 
 struct gpio_regulator_platform_data
-- 
1.7.9.5

